src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -106,4 +105,28 @@ struct Normalized : db::ComputeTag {
   }
   using argument_tags = tmpl::list<Tag, Magnitude<Tag>>;
 };
+
+/// \ingroup TensorGroup
+/// \brief Compute square root of the Euclidean magnitude of a rank-0 tensor
+///
+/// \details
+/// Computes the square root of the absolute value of the scalar.
+template <typename DataType>
+Scalar<DataType> sqrt_magnitude_of_scalar(
```

Comment:
- Why is this function in the `Tags` namespace?
- why `_of_scalar`? That's given just by the argument type, so I don't think that's necessary.

Reply:
agreed, fixed.